packages/core-suggest/core-suggest.js

```diff
@@ -136,34 +274,73 @@ function onMutation (self) {
   self._observer.takeRecords() // Empty mutation queue to skip mutations done by highlighting
 }
 
+/**
+ * Handle input event in connected input
+ * Performs filtering of core-suggest items
+ * Dispatches event
+ *  * `suggest.filter`
+ * @param {CoreSuggest} self Core suggest element
+ * @param {InputEvent} event
+ * @returns {void}
+ */
 function onInput (self, event) {
-  if (event.target !== self.input || !dispatchEvent(self, 'suggest.filter') || onAjax(self)) return
+  if (event.target !== self.input || !dispatchEvent(self, 'suggest.filter') || onAjax(self) || self.hasAttribute('filter-disabled')) return
   const value = self.input.value.toLowerCase()
   const items = self.querySelectorAll('a,button')
 
   for (let i = 0, l = items.length; i < l; ++i) {
     toggleItem(items[i], (items[i].value || items[i].textContent).toLowerCase().indexOf(value) === -1)
   }
+
+  if (items.length > 0) {
+    const visibleItems = queryAll('[tabindex="-1"]:not([hidden])', self)
+    if (visibleItems.length === 0) {
+      // All items have been hidden by filter
+      notifyResultsEmpty(self)
+    } else if (visibleItems.length < items.length) {
+      notifyResultCount(self, visibleItems.length)
+    }
+  }
 }
 
+/**
+ *
+ * @param {CoreSuggest} self Core suggest element
+ * @param {KeyboardEvent} event
+ * @returns {void}
+ */
 function onKey (self, event) {
   if (!self.contains(event.target) && self.input !== event.target) return
   const items = [self.input].concat(queryAll('[tabindex="-1"]:not([hidden])', self))
-  let { keyCode, target, item = false } = event
-
-  if (keyCode === KEYS.DOWN) item = items[items.indexOf(target) + 1] || items[0]
-  else if (keyCode === KEYS.UP) item = items[items.indexOf(target) - 1] || items.pop()
-  else if (self.contains(target)) { // Aditional shortcuts if focus is inside list
-    if (keyCode === KEYS.END || keyCode === KEYS.PAGEDOWN) item = items.pop()
-    else if (keyCode === KEYS.HOME || keyCode === KEYS.PAGEUP) item = items[1]
-    else if (keyCode !== KEYS.ENTER) items[0].focus()
+  let { key, target, item = false } = event
+
+  if (key === KEY.DOWN || key === KEY.DOWN_IE) {
+    item = items[items.indexOf(target) + 1] || items[0]
+  } else if (key === KEY.UP || key === KEY.UP_IE) {
+    item = items[items.indexOf(target) - 1] || items.pop()
+  } else if (self.contains(target)) {
+    // Aditional shortcuts if focus is inside list
+    if (key === KEY.END || key === KEY.PAGEDOWN) {
+      item = items.pop()
+    } else if (key === KEY.HOME || key === KEY.PAGEUP) {
+      item = items[1]
+    } else if (key !== KEY.ENTER) {
+      items[0].focus()
+    }
   }
 
-  setTimeout(() => (self.hidden = keyCode === KEYS.ESC)) // Let focus buble first
-  if (item || keyCode === KEYS.ESC) event.preventDefault() // Prevent leaving maximized safari
+  setTimeout(() => (self.hidden = (key === KEY.ESC || key === KEY.ESC_IE))) // Let focus buble first
```

Comment:
> [2] In Internet Explorer (tested on release 9 and 11) and Firefox 36 and earlier, the Esc key returns "Esc" instead of "Escape".

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values#ui_keys

Reply:
Ah, du har byttet til event.key i stede for event.keyCode hvor ID er det samme :D

Reply:
Tenkte det var greit å oppdatere bort fra deprecation-flaggene til keyCode